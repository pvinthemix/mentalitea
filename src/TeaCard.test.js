// import React from 'react';
// import TeaCard from './TeaCard';
// import { shallow } from 'enzyme';

// describe('TeaCard', () => {
//   let wrapper;

//   beforeEach(() => {
//     wrapper = shallow(
//       );
//   });

//   it('should ', () => {

//   });