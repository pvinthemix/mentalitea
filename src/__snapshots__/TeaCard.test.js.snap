// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeaCard should display tea card and tea image 1`] = `
<li
  className="tea-card"
  onClick={[Function]}
>
  <h2
    className="tea-name-unexpanded"
  >
    Irish Breakfast
  </h2>
  <img
    alt=""
    className="tea-image-unexpanded"
    src="Irish Breakfast.png"
  />
</li>
`;

